pymc/sampling/mcmc.py

```diff
@@ -303,8 +313,11 @@ def _sample_external_nuts(
             random_seed=random_seed,
             initvals=initvals,
             model=model,
+            keep_untransformed=nuts_sampler_kwargs.get("keep_untransformed", False),
```

Comment:
why can't we just `**nuts_sampler_kwargs`?

Reply:
We could. This seemed a little more targeted, especially since there are only a few relevant arguments.

Reply:
I think that'd be better, because:
1. If there are new kwargs added or ones we didn't think of, we support it.
2. We need to track the default kwargs here which might also change.

Reply:
And we can use the same approach for `nutpie` which has different kwargs.